tcp/tcp.py

```diff
@@ -99,8 +99,10 @@ def __process_requests(self, client_socket, client_addr):
         This function serves the control socket's coming requests.
         """
         reinstall_file = list()
-        with open('reinstall.txt') as fp:
+        with open('reinstall.txt', "r+") as fp:
             reinstall_file = fp.read()
+            fp.seek(0)
```

Comment:
no, we don't want to clear the file here since we might not be getting the request from the file we are trying to reinstall.  you want to move the truncate code further down to where we actually match on the IP we want to reinstall